tests/cloudferrylib/os/identity/test_keystone.py

```diff
@@ -115,6 +121,10 @@ def setUp(self):
         src_keystone.try_get_user_by_name.side_effect = self.mock_user_get
         dst_keystone.try_get_user_by_name.side_effect = self.mock_user_get
 
+    def test_get_client_check(self):
```

Comment:
As, you had asked to add a unit test for this method, I thought to add unit test for it. But I think this will be covered automatically. As get_client will be called whenever a keystone_client will be called from keystone module.